front/src/common/ResizableSection.tsx

```diff
@@ -15,6 +15,10 @@ const ResizableSection = ({
   const [baseHeight, setBaseHeight] = useState(height);
   const [sectionHeight, setSectionHeight] = useState(baseHeight);
 
+  useEffect(() => {
+    setSectionHeight(height);
+  }, [height]);
```

Comment:
Why do you need this useEffect ? 

Hum I'm not very sure of how this component works :thinking:
- onResize, we update the sectionHeight and the height => the update of the height will fire this useEffect and update the sectionHeight again
  Maybe we don't need the sectionHeight and use directly the height ?
- I'm not sure what baseHeight is used for :thinking: (but maybe this is out of scope of this PR)